server/src/curfu/routers/utilities.py

```diff
@@ -38,10 +38,10 @@ def get_mane_transcripts(request: Request, term: str) -> dict:
     """
     normalized = request.app.state.fusor.gene_normalizer.normalize(term)
     if normalized.match_type == gene_schemas.MatchType.NO_MATCH:
-        return {"warnings": [f"Normalization error: {term}"]}
-    if not normalized.gene_descriptor.gene_id.lower().startswith("hgnc"):
-        return {"warnings": [f"No HGNC symbol: {term}"]}
-    symbol = normalized.gene_descriptor.label
+        return {"warnings": [f"Normalization error: {term}"], "transcripts": None}
+    if not normalized.gene.id.replace("normalize.gene.", "").lower().startswith("hgnc"):
```

Comment:
I think this can be simplified?
```suggestion
    if not normalized.normalized_id.startswith("hgnc"):
```